tests/test_convert.py

```diff
@@ -18,41 +18,37 @@ def test_conversion(input_step_file):
     output_filename_stem = output_dir / input_step_file.stem
 
     # creates the config file contents
-    template = (
-        "[Files]\n"
-        "title = 'Input Test'\n"
-        f"stepFile = {input_step_file.resolve()}\n"
-        f"geometryName = {output_filename_stem.resolve()}\n"
-        "outFormat = ('mcnp', 'openMC_XML')\n"
-        "[Parameters]\n"
-        "compSolids = False\n"
-        "volCARD = False\n"
-        "volSDEF = True\n"
-        "voidGen = True\n"
-        "dummyMat = True\n"
-        "minVoidSize = 100\n"
-        "cellSummaryFile = False\n"
-        "cellCommentFile = False\n"
-        "debug = False\n"
-        "simplify = no\n"
-        "[Options]\n"
-        "forceCylinder = False\n"
-        "splitTolerance = 0\n"
-        "newSplitPlane = True\n"
-        "nPlaneReverse = 0\n"
-    )
-
-    with open(output_dir / "config.ini", mode="w") as file:
-        file.write(template)
+    template = {
```

Comment:
```[Parameters]```, ```[Options]``` and the other sections identifiers are still required in the external config file when the method ```SetConfiguration``` (renamed from ```SetOptions``` ) is used to pass the parameters values. 
In the  tests/test_convert.py I changed the mode to pass the parameters, instead of writing a file and use the  ```SetConfiguration``` method, all parameters are stored in a dictionnary and passed with the method ```set```. This method is able to identify any valid parameter name (of any section) and assign its value to the corresponding variable/attribute.